Detections/SlackAudit/SlackAuditMultiple archivedFilesUploadedInShortTimePeriod.yaml

```diff
@@ -0,0 +1,32 @@
+id: 995fada8-9101-4b4b-a75d-e11bcb80e9be
+name: SlackAudit - Multiple archived files uploaded in short time period
+description: |
+  'This query helps to detect fact of uploading multiple archived files in short time period.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SlackAuditAPI
+    dataTypes:
+      - SlackAudit_CL
+queryFrequency: 30m
```

Comment:
30m for a low level alert is not recommended.  I recommend changing to 1d and you can use bin(TimeGenerated, 30m) to get a count across the 1 day so you can check against the threshold.

Reply:
Fixed, Pls check